src/data/html-quiz.ts

```diff
@@ -1525,5 +1525,18 @@ const htmlQuiz = [
       "The <i> tag is used to make text italic without adding extra emphasis. It is commonly used for text that is set off from normal prose, like foreign words or thoughts. ",
     Link: "https://devdocs.io/html/element/i",
   },
+  {
+    Question:
+      "In HTML, responsive text size can be set with a" "vw" "unit, which means the",
+    Answer: "<viewport width>",
+    Distractor1: "<versatile width>",
+    Distractor2: "<viewing wide>",
+    Distractor3: "<view length>",
+    Explanation:
+      "The text size can be set with a vw unit, which means the viewport width.Viewport is the browser window size.Example 1vw = 1% of viewport width"
+         Link: "https://www.w3schools.com/html/html_responsive.asp",
```

Comment:
Please fix this syntax error so it can be ready for review

Reply:
I have tried to fix the mistakes can you please check , I am a first time contributor therefore making mistakes .